engine/common/host.c

```diff
@@ -914,6 +914,13 @@ static void Host_Userconfigd_f( void )
 	Mem_Free( t );
 }
 
+#if !XASH_DEDICATED
+static void Host_WriteConfig_f( void )
```

Comment:
I don't think you have to add _f function for it, as Host_WriteConfig function matches needed signature to be a command.

Other than that, I'm fine with the changes.

Reply:
Initially I wanted to add a parameter to specify config name, but I'm not sure if that's a good idea and what's the best way to do it